src/Compilers/VisualBasic/Portable/LanguageVersion.vb

```diff
@@ -13,6 +13,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         VisualBasic12 = 12
         VisualBasic14 = 14
         VisualBasic15 = 15
+        VBvNext = Integer.MaxValue
```

Comment:
Note: I've used `VBvNext` as whilst developing the feature, used to indicate "prototype" language features. Also `VisualBasic15` was pushed into repo, added it to so it minimised the effects it had on my work.